src/arrayReverse.test.js

```diff
@@ -1,20 +1,35 @@
 'use strict';
 
-describe(`Function 'arrayReverse':`, () => {
+describe('arrayReverse', () => {
   const arrayReverse = require('./arrayReverse');
 
-  it(`should be declared`, () => {
+  it('should be declared', () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it('should reverse the order of the words in the array', () => {
```

Comment:
It would be more accurate to call it 'should reverse the order of the letters in each word in the array'